decktape.js

```diff
@@ -102,6 +102,16 @@ parser.script('decktape').options({
     transform : parseRange,
     help      : 'Range of slides to be exported, a combination of slide indexes and ranges (e.g. \'1-3,5,8\')',
   },
+  headless : {
+    default : true,
```

Comment:
Could you change it to the new default `new`, as Decktape has been updated to use https://developer.chrome.com/articles/new-headless/#new-headless-in-puppeteer.